native/src/components/OrganizationContentInfo.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ReactElement } from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import styled from 'styled-components/native'
+
+import { OrganizationModel } from 'api-client'
+
+import HighlightBox from './HighlightBox'
+import Link from './Link'
+import SimpleImage from './SimpleImage'
+import Text from './base/Text'
+
+const Thumbnail = styled(SimpleImage)`
+  height: 80px;
+`
+
+const Box = styled(HighlightBox)`
+  display: flex;
+  font-family: ${props => props.theme.fonts.native.decorativeFontRegular};
+  margin-bottom: 16px;
+`
+
+const Column = styled.View`
+  display: flex;
+  flex-direction: column;
+`
+
+const OrganizationContent = styled.Text`
+  font-family: ${props => props.theme.fonts.native.decorativeFontBold};
+  padding: 16px 0 8px;
+`
+
+type OrganizationContentInfoProps = {
+  organization: OrganizationModel
+}
+
+const OrganizationContentInfo = ({ organization }: OrganizationContentInfoProps): ReactElement => {
+  const { t } = useTranslation('categories')
+  return (
+    <Box $padding>
+      <Thumbnail source={organization.logo} />
```

Comment:
I am not able to get this left aligned, can someone help?

Reply:
If you give it a width, you can do an `align-self: flex-start;`. But I'm not sure if we have a fixed width?

Reply:
Yes, sadly we don't have a fixed width :(

Reply:
Oh. That's sth I also didn't know. For lunes I already did sth. similar, but assumed the aspect ratio to be a fixed value. https://github.com/digitalfabrik/lunes-app/pull/441

Reply:
i'm not sure what you mean @sarahsporck . When you define a height or a width and want the calculated other parameter like in this case the width for a proposed height you need to give an aspect ratio.
You gave a `100%` value so how could you calculate a value out of that with an aspect ratio.

Reply:
Jep. Ahh. I think I get the problem. When the aspect ratio is too big the image will "grow" vertically so that it shifts to the right. What confused me was that the aspect ratio actually does not change tha aspect ration of the image, but of the Wrapper, as the image has `resizeMode=contain` it will keep the same aspect ratio, and only changes in scale, while the wrapper actually changes it's aspect ratio. So long story short we need to calculate the width to set the aspect ratio of the wrapper to the same ratio of the image. Than it is can be aligned correctly.

Reply:
Thanks, works like a charm! :rocket: